compute_endpoint/tests/integration/endpoint/executors/high_throughput/test_htex_regression.py

```diff
@@ -0,0 +1,86 @@
+import concurrent.futures
+import queue
+import random
+import uuid
+
+import pytest
+from globus_compute_common import messagepack
+from globus_compute_endpoint.engines import HighThroughputEngine
+from globus_compute_sdk.serialize import ComputeSerializer
+from tests.utils import double, ez_pack_function
+
+
+@pytest.fixture
+def engine():
+    ep_id = uuid.uuid4()
+    engine = HighThroughputEngine(
+        label="HTEXEngine",
+        heartbeat_period=1,
+        worker_debug=True,
+    )
+    results_queue = queue.Queue()
+    engine.start(endpoint_id=ep_id, run_dir="/tmp", results_passthrough=results_queue)
+
+    yield engine
+    engine.shutdown()
+
+
+@pytest.mark.skip()
+def test_engine_submit_internal(engine):
```

Comment:
It is perhaps odd to immediately skip a newly defined test.  Should this be fixed-up and unskipped?  Or removed entirely?

Reply:
Removed this test. The following test indirectly covers this.